test/input_gen/genLayerTests.py

```diff
@@ -48,8 +48,13 @@ def inspect_file(file_name):
     record_single(fc, (3, 1, 1, 10), "fc_golden_plain")
     fc = K.layers.Dense(4)
     record_single(fc, (1, 1, 1, 10), "fc_golden_single_batch")
-
-# inspect_file("fc_golden.nnlayergolden")
-
-
+    bn = K.layers.BatchNormalization()
+    record_single(bn, (2, 4, 2, 3), "bn_golden_channels_training", {"training": True})
+    ## @todo add test for inference
+    record_single(bn, (2, 4, 2, 3), "bn_golden_channels_inference", {"training": False})
+    bn = K.layers.BatchNormalization()
```

Comment:
We can remove this line I guess?

Reply:
input shape is different so this has to be initialized again